test/AotCompile/model_loader_lib.py

```diff
@@ -313,6 +313,30 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
     )
 
 
+def round_loader() -> TorchLoaderOutput:
+    class Round(torch.nn.Module):
```

Comment:
Perhaps call this `RoundEven` and the loader `round_even_loader` to disambiguate.